Dockerfile

```diff
@@ -61,17 +63,23 @@ RUN pip3 install -U pip \
   && pip3 install git+https://github.com/ethereum/pyethereum.git@3d5ec14032cc471f4dcfc7cc5c947294daf85fe0 \
   && rm -r ~/.cache/pip
 
-COPY solc /usr/bin/
-RUN chmod +x /usr/bin/solc
+RUN curl -o solc -L https://github.com/ethereum/solidity/releases/download/v0.4.24/solc-static-linux \
+  && mv solc /usr/bin/ \
+  && chmod +x /usr/bin/solc
```

Comment:
A little redundance.
```bash
RUN curl -o /usr/bin/solc \
        -L https://github.com/ethereum/solidity/releases/download/v0.4.24/solc-static-linux \
    && chmod +x /usr/bin/solc
```
Never mind. :smile: 

Many years ago, when I tried using linux first time, the first suggest I got is:
_You can control any files without `cd` into where they are._